README.md

```diff
@@ -12,32 +12,33 @@ It started as a [cribbbage hand scorer](https://bicyclecards.com/how-to-play/cri
 1. Go get this project
 
 ```bash
-go get github.com/joshprzybyszewski/cribbage
+go get -u github.com/joshprzybyszewski/cribbage
 ```
 
-2. Vendor the dependencies
+2. Install everything you need
 
 ```bash
-GO111MODULE=on go mod vendor
+make install
```

Comment:
FYI this only works for macOS out of the box (windows doesn't have make by default)

Reply:
who wants to run this on windows tho? :# ;)